Client/Frontend/Browser/TabManager.swift

```diff
@@ -118,5 +118,6 @@ class TabManager {
         }
         
         assertionFailure("Tab not in tabs list")
+        return -1
```

Comment:
Note that assertions are only enabled in debug builds. Hopefully this isn't a bug that will make release!